### ISSUE TITLE
Vulnerability: code injection

### ISSUE THREAD

**Issue by h2oa** [scikit-image/scikit-image#7413]
### Description:

Hi,

I have reported a dangerous security vulnerability. You can connect to admin of platform huntr.com see details at [link](https://huntr.com/bounties/e125c410-0dad-4408-a818-3828ab6fd755) (private link)

**Comment by h2oa** [comment]
Hi @lagru , I sent a mail report to security@tidelift.com but never receive the reply :(

**Comment by lagru** [comment]
Hmm, that definitely shouldn't happen. How long ago did you send the email? We checked with Tidelift and they recommend to re-send the message. It might have been flagged as spam accidentally.

**Comment by h2oa** [comment]
Hi @lagru , I sent the email on 2024/04/30. I have just resend it, thank you!

**Comment by lagru** [comment]
Hey @h2oa, we received the email. Like this issue, it looks like it's asking us to sign up with huntr? To be honest, we haven't had good experiences with huntr in the past with regards to spam. Not saying that is the case with your report, :) but could we ask you to send the actual report directly to [security@tidelift.com](mailto:security@tidelift.com) at Tidelift without asking us to sign up? Thanks!

Since we don't have contact address from you, alternatively, you may also reach out to @stefanv or me directly via the contact information provided on [Stéfan's keybase](https://keybase.io/stefanvdwalt) or [my website](https://grueter.dev/contact). In both cases you will find an email and PGP keys to send us an encrypted message.

I'm attaching my PGP key here as well for verification purposes.

<details><summary>Details</summary>

```plain
-----BEGIN PGP PUBLIC KEY BLOCK-----

mQENBF5FMlMBCADGBQVSQQUYgLnpel82JNNsY9nKSKfTHYM5OyiQJAkvsiLneEze
VMzCj7IgsfSBFXxb2MYGUYT7ZnxEOG53sVOTkt9M0c5l7M2hKpxyTcSp2HrynO9V
1c7Qec+jCdpZECtCEk3BT4Zmop6tbQIsOCunAtkby9lxt9PIS+7jStfN9ByE7ehW
Cf2Sim2BZvCqfnEnEmyBrmE3j2BeBnDvzSQ8pvWAOdRMKcrZZg58Jt2TNlYXRfoh
/VDcktBZbvMHWFQlAtq7JKUA16rz9+QJwLr+ry97J5NC34vr8nQ/vvhWxuSkZsmN
tlQQOJBS//U1spB0VzCwGRqrVGoLsM3cyGhNABEBAAG0IkxhcnMgR3LDvHRlciA8
Y29udGFjdEBncnVldGVyLmRldj6JAVQEEwEIAD4WIQR7dIK6L6foW8AJ2MmGH0Rs
XL8XPgUCZTMN3gIbAwUJCfcQDgULCQgHAgYVCgkICwIEFgIDAQIeAQIXgAAKCRCG
H0RsXL8XPgO0CACB2qBMNeoiPaOpctc02e51juTdKKn/n5ZfjBJJtFd0mgTr5Uof
P55IjErlSbmakoTuZGLw+iD44uWc/O1eJkL/ElEAF9Al2i0kKqs+VOa/cI10Fsa4
Qz4e++yuPvhWw2Fmo2DQv2042SNhwYw/HCWy8MiXCF3tBatVThiPoQSYUyYYDzhe
anrqhZY8mR/9mGa5zicNhRuD6LyFlHBavxekuGJvz3YipTnVNFd1YqV2VegK1tgQ
4YXE6a7l9nB4QTpYLfa7p7k8u/xZmtpeVUnXSe5jy7cUJK1g4tqmunIpWNwGrK0E
JbzYEVFJjOdw/8AdMgePqgpYZYiVykw7Z/DOtCBMYXJzIEdyw7x0ZXIgPGxhZ3J1
QG1haWxib3gub3JnPokBVwQTAQgAQQIbAwULCQgHAgYVCgkICwIEFgIDAQIeAQIX
gAUJCfcQDhYhBHt0grovp+hbwAnYyYYfRGxcvxc+BQJlMw7bAhkBAAoJEIYfRGxc
vxc+Qw0IAKOT8Tfy7MpJ8TTVzAgHMSapc07zlPU4F14nyIkAOpzueEAWUxB/BIBZ
u57uYPstQXxU9j8VUnfi4smDGPtDPRp4Cv/OmS/12VfDTUKahcnxK/m73/Xu00NT
D+j+Pnx3TCG4UB3HAzLiVS10mFeAOfTiHtfFIP8AlboMG1s7rqYBPiB74sWrsmWL
Kn0wwj1IawlMgiqdvo0Zwby1hADkHQximGKVmGaEPxfH77dsoQAWwUWKNT3r4iSv
2RpHUD/DZdkx+N2tYA5xfukhDqUv74kfio9RqjhIEb3m2mwfbQQfBPVh0O3iO1OD
uSnEs91L/yhrAUnG0n1LDU4edHMHTfi5AQ0EXkUyUwEIAJ45tkHS9zNvYUZUJR9G
Di6AVOmElEWL1UAmwQlET4LVmf4p3FLiKuwGW5ME4GE3KtVLQIMF4bTQsMRWjF6F
ku9ndyTbKTmApBSh4HS6j4u7MBZKlTfddukG17e9e52AqEPmkp9zibBcb6LH8EoH
tta2jWOvDKHmLPW1eSnoZ4PTKzQlwUENUKpfUcyFWC0wByV00lMrrl8/y+MCzoWd
X5x3wtPfeDAmJWTdMp2iXK8LR//JzWx2b1GeZhqqohK1l5NhsKQ03AI4tJbSqstQ
RTj7I86N0UVBJHL2h7j/Fbe3gRx129LZ9IFJFkA1XR1xs6DhA5oqqN3/ox9Ewkpq
eOMAEQEAAYkBPAQYAQgAJgIbDBYhBHt0grovp+hbwAnYyYYfRGxcvxc+BQJkedvL
BQkJ9xB4AAoJEIYfRGxcvxc+gZ8H+QHZLwubToO8Wwyd2JQPM6Sinu78o6fKMm/X
P0GmLjjemWyCxFquoewxnC9MOdI6OxGckr/3YOWL8BwJmNkILji4Cx4ZJbMxvEyP
vJ8aYUvZOh0v468QXiJao/qUzg2DmYwvJ+ZtruzvrrDZiiisxOlrI//b31W90ZFp
zA/aMjrDCvioBgBxTm/vY9EIeM3sMKwH2MZhSb8WWxIkFqK0ZBC9T2ZAFZkFnz9m
A8WuUQyquVq+7UMMrZ2bFcoqT6G9LVWJHcto7x5Len584FICt88V7N8A3IAYGYCC
zorm8ei4PBTHtA4LD4YFVjYLjlpOGP0T+GQ3mv+AJEa4FO/VGd4=
=lH8h
-----END PGP PUBLIC KEY BLOCK-----

```

</details>

**Comment by lagru** [comment]
Fyi, if this is a valid security concern, we can always sign up after the fact so you can receive a reward on https://huntr.com/bounties

**Comment by h2oa** [comment]
Hi @lagru,

Thank you for your feedback, I have written a new email to security@tidelift.com and attached the details of the vulnerability report. Please consider it.

Best regards,
@h2oa

**Comment by lagru** [comment]
Thanks, we've received it and are currently looking into it. :pray:

**Comment by lagru** [comment]
Addressed in #7415.
